cmd/analyze.go

```diff
@@ -153,65 +228,95 @@ This will create a single host for hostnames that resolve to the same IPs`,
 				}
 			}
 
-			if len(hosts) == 0 {
+			hostLen := len(hosts)
+			msgs := []string{}
+			if hostLen == 0 {
+				if update {
+					continue
+				}
 				// Still no host, lets create a new one
-				fmt.Printf("[+] Creating new service %s\n", domain)
+				msgs = append(msgs, fmt.Sprintf("[+] Creating new service %s\n", domain))
 				h = host.InitHost(filepath.Join("hosts", domain))
-			} else {
+			} else if hostLen == 1 {
 				h = hosts[0]
-				if h.Metadata.Name != domain {
-					fmt.Printf("[+] Adding domains to %s from %s\n", h.Metadata.Name, domain)
-				} else {
-					fmt.Printf("[+] Updating existing %s\n", domain)
+			} else {
+				fmt.Printf("[+] more than one host (%s) found for %s", hostLen, domain)
```

Comment:
Should be `fmt.Printf("[+] more than one host (%d) found for %s", hostLen, domain)`